wiki/fr/fr-Getting-Started.md

```diff
@@ -58,11 +60,11 @@ Vous trouverez plus d'informations sur comment éviter de vous écouter (et pour
 
 ## Comment Jamulus fonctionne (principe de base)
 
-**Voici comment des millions de personne utilisent Jamulus pour jouer, chanter et apprendre la musique avec d'autres personnes en temps réel à travers le monde :**
+**Voici comment des millions de personnes utilisent Jamulus pour jouer, chanter et enseignent la musique avec d'autres personnes en temps réel à travers le monde :**
 
-![Un schéma pour expliquer le principe de fonctionnement de Jamulus](https://user-images.githubusercontent.com/44293594/99189852-b2351180-2763-11eb-96fc-660f5038a83c.png)
+![vue d'ensemble](https://user-images.githubusercontent.com/9108457/99901946-9200d780-2cba-11eb-8167-e20ae2621102.png)
 _Jamulus fonctionne sur le principe clients/serveurs. Le son de chacun est envoyé au serveur, qui les mixe et les traite. Ensuite, le son est envoyé à chaque client. Si un serveur est public et enregistré sur un serveur central, ses informations seront diffusées à tous les clients._
 
 ## Notes
-[^1]: [Tous les détails ici](Network-Requirements){: target="_blank" rel="noopener noreferrer"}. Si vous avez une connexion avec 10Mbits descendants et 1Mbps montants, vous n'aurez sûrement aucun problème de bande-passante.
-[^2]: Les casques sans-fil Bluetooth basiques ont généralement beaucoup trop de latence. C'est pourquoi, il est préférable d'utiliser un casque ou des écouteurs filaires. En particulier si vous chantez ou jouez d'un instrument accoustique, pour être sûr de jouer en même temps n'utilisez qu'un casque ou des écouteurs filaires. (voir ces [notes](Getting-Started#having-trouble-cant-keep-in-time) pour de plus amples informations).
+[^1]: [Tous les détails ici](Network-Requirements){: target="_blank" rel="noopener noreferrer"}. Si vous avez une connexion avec 10Mbits descendant et 1Mbps montant, vous n'aurez sûrement aucun problème de bande-passante.
+[^2]: Les casques sans-fil Bluetooth basiques ont généralement beaucoup trop de latence. C'est pourquoi, il est préférable d'utiliser un casque ou des écouteurs filaires. En particulier si vous chantez ou jouez d'un instrument accoustique, pour être sûr de jouer en rythme utilisez uniquement des casques filaires (voir ce [paragraphe à ce propos](Getting-Started#having-trouble-cant-keep-in-time) pour de plus amples informations).
```

Comment:
@Saimyx the internal link must be updated.
```md
[paragraphe à ce propos](Getting-Started#vous-avez-des-problèmes--vous-n-arrivez-pas- à-être-en-rythme-)
```
I used this sentence for the linked title to not have to deal with the « ' » (not sur the code above works)
```md
## Vous avez des problèmes ? Vous ne parvenez pas à jouer en rythme ?
```
With this, the link must be
```md
[paragraphe à ce propos](Getting-Started#vous-avez-des-problèmes--vous-ne-parvenez-pas-à-jouer-en rythme-)
```